index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+    <html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="stylesheet" type="text/css" href="main.css">
+        <title>Sing in project</title>
+    </head>
+
+    <body>
+        <div id="container">
```

Comment:
`OPTIONAL` 

Consider using `main` instead, this shows the dominant content of the `<body>` of a document. More on `<main>` [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) :star2: